op-challenger/fault/caller.go

```diff
@@ -0,0 +1,79 @@
+package fault
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type FaultCaller struct {
+	*bindings.FaultDisputeGameCaller
+
+	log log.Logger
+}
+
+func NewFaultCaller(fdgAddr common.Address, client *ethclient.Client, log log.Logger) (*FaultCaller, error) {
+	caller, err := bindings.NewFaultDisputeGameCaller(fdgAddr, client)
+	if err != nil {
+		return nil, err
+	}
+	return &FaultCaller{
+		caller,
+		log,
+	}, nil
+}
+
+// LogGameInfo logs the game info.
+func (fc *FaultCaller) LogGameInfo() {
+	fc.LogGameStatus()
+	fc.LogClaimDataLength()
```

Comment:
Could I suggest we reduce this down to a single log line. So we get something like `Game status addr=0x1234 claims=10 status=In Progress`. Makes it much easier to find and read the log lines.  The address isn't really important now but may as well include it since we'll eventually support multiple games and it's easy to include now.

I'd just not log the status if either piece of info can't be retrieved for simplicity.

Reply:
100% always happy to simplify